lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -236,6 +236,25 @@ def _get_congestion_control_config(service_mconfig):
 
     return True
 
+def _get_converged_core_config(service_mconfig):
+    """
+    Retrieves enable_converged_core config value, it it does not exist
+    it defaults to True.
+    Args:
+        service_mconfig:
+
+    Returns: converged core flag
+    """
+    enable_converged_core = get_service_config_value(
```

Comment:
The formatting looks a bit off here. Can you follow the steps outlined here to format this file?: https://github.com/magma/magma/tree/master/lte/gateway/docker/python-precommit